addons/binding/org.openhab.binding.homematic/ESH-INF/thing/bridge.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="homematic"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Homematic Bridge -->
+	<bridge-type id="bridge">
+		<label>Homematic Bridge</label>
+		<description>The Homematic bridge represents a Homematic gateway</description>
+
+		<config-description>
+			<parameter name="gatewayAddress" type="text" required="true">
+				<context>network_address</context>
+				<label>Gateway Address</label>
+				<description>Network address of the Homematic gateway</description>
+			</parameter>
+            <parameter name="gatewayType" type="text">
+                 <label>Gateway Type</label>
+                 <description>Hint for the binding to identify the gateway type (default = auto)</description>
+                 <limitToOptions>true</limitToOptions>
+                 <default>auto</default>
+                 <options>
+                     <option value="auto">Auto</option>
+                     <option value="ccu">CCU</option>
+                 </options>
+            </parameter>
+			<parameter name="callbackHost" type="text">
+				<context>network_address</context>
+				<label>Callback Network Address</label>
+				<description>Callback network address of the openHAB server, default is auto-discovery</description>
+			</parameter>
+			<parameter name="callbackPort" type="integer">
+				<label>Callback Port</label>
+				<description>Callback port of the openHAB server, default is 9125 and counts up for each additional bridge</description>
+			</parameter>
+			<parameter name="aliveInterval" type="integer">
+				<label>Alive Interval</label>
+				<description>The interval in seconds to check if the communication with the Homematic gateway is still alive. 
+				If no message receives from the Homematic gateway, the RPC server restarts (default = 300)</description>
```

Comment:
can you declare the default value as a tag and remove it from the description? (Also for all following parameters)